src/index.js

```diff
@@ -437,6 +437,19 @@ export default class AttachesTool {
     this.nodes.wrapper.appendChild(fileInfo);
 
     if (file.url !== undefined) {
+      // Make the whole box clickable in read-only mode
+      if (this.readOnly) {
+        this.nodes.wrapper.setAttribute('role', 'button');
+        this.nodes.wrapper.addEventListener('click', (event) => {
+          // Don't trigger if clicking on the actual download link
+          if (event.target.closest(`.${this.CSS.downloadButton}`)) {
+            return;
+          }
+          // Open file in new tab/window
+          window.open(file.url, '_blank');
+        });
+      }
```

Comment:
I'd suggest to use `<a>` as a file wrapper element in readonly. And `span` for downloadIcon.

It will be more performant for large articles because we won't add redundant click handlers. 

Also, please, add hover effect (change `--color-line` to `#e7e8e9`) on wrapper in readonly.